sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -253,9 +260,8 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
    */
   async renewSessionLock(options?: OperationOptions): Promise<Date> {
     this._throwIfReceiverOrConnectionClosed();
-
+    this._throwIfMessageSessionDoesntExist();
```

Comment:
Do we really need to check this at all? Since these operations are on the management link, they technically don't need the `messageSession`. We had this in Track 1 because `createSessionReceiver()` does not guarantee getting a lock on the session. Since that has now changed, can we skip this check altogether for all of the management link operations? I would expect the service to throw an error if the lock was lost and the operation cannot be performed.

This would apply to peek, set/get state, receiveDeferredMessages as well